### ISSUE TITLE
Tagging Collection Level Files

### ISSUE THREAD

**Issue by hoodriverheather** [Labcas-NIST/Labcas-ui#149]
@yuliujpl I know we briefly discussed this a long time ago. Would it be possible to allow users to tag items as CollectionLevel in the fiile metadata and then assign the hierarchy they would be placed in as part of the file level metadata? 
Here's the draft metadata:
<google-sheets-html-origin><style type="text/css"></style>
DRAFT: |   |   |   |   |  
-- | -- | -- | -- | -- | --
CollectionLevelFile | TrueFalse | Capture required CoreFile metadata. Need a way to define the CollectionLevel structure. |   |   | Selecting "Yes" indicates that this file pertains to the entire Collection (as opposed to a dataset within the collection; e.g., a study design document or benchmarking file) and should be highlighted and placed outside of and above the file list box in the LabCAS UI.
  |   |   |   |   |  
CollectionLevelFileHierarcy |   | could be captured something like: Benchmark/Documentation |   |   |

**Comment by yuliujpl** [comment]
Yup! The way to do it now is to have the folder name be the same as the collection id/name. Adding that metadata would be helpful too, and I can have my parsers move files to the collection level file directory!
________________________________
From: hoodriverheather ***@***.***>
Sent: Thursday, October 3, 2024 12:22 PM
To: Labcas-NIST/Labcas-ui ***@***.***>
Cc: Liu, David (US 3902-Affiliate) ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] [Labcas-NIST/Labcas-ui] Tagging Collection Level Files (Issue #149)


@yuliujpl<https://urldefense.us/v3/__https://github.com/yuliujpl__;!!PvBDto6Hs4WbVuu7!LV28Dv2NVWkOqhxSX7RaSsWhk_bxN_Y1vLjcWEtb9WdtClDwuh75W7UCnI_pUOQbff8h1-BaoM5bJ1yC7mZWHJcBzmYP6Q$> I know we briefly discussed this a long time ago. Would it be possible to allow users to tag items as CollectionLevel in the fiile metadata and then assign the hierarchy they would be placed in as part of the file level metadata?
Here's the draft metadata:
<style type="text/css"></style>

DRAFT:
CollectionLevelFile     TrueFalse       Capture required CoreFile metadata. Need a way to define the CollectionLevel structure.                 Selecting "Yes" indicates that this file pertains to the entire Collection (as opposed to a dataset within the collection; e.g., a study design document or benchmarking file) and should be highlighted and placed outside of and above the file list box in the LabCAS UI.

CollectionLevelFileHierarcy             could be captured something like: Benchmark/Documentation

**Comment by hoodriverheather** [comment]
@yuliujpl Awesome! can you give me an example of the collection id/name so i can include that in the dictionary?

**Comment by yuliujpl** [comment]
Ah, it's the exact ID of that colleciton, so for NIST Genome, it's: Genome_Editing_Consortium!
________________________________
From: hoodriverheather ***@***.***>
Sent: Thursday, October 3, 2024 12:59 PM
To: Labcas-NIST/Labcas-ui ***@***.***>
Cc: Liu, David (US 3902-Affiliate) ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [Labcas-NIST/Labcas-ui] Tagging Collection Level Files (Issue #149)


@yuliujpl<https://urldefense.us/v3/__https://github.com/yuliujpl__;!!PvBDto6Hs4WbVuu7!IZvlxjwSbg_Vb1gRERXhpl86-8qdnqhSHLBcHScUu8bVTCQ6osCFC05r6JZTMBXUAyClN8HJCE_uHC-V_JGT07fcX9rTVQ$> Awesome! can you give me an example of the collection id/name so i can include that in the dictionary?

**Comment by hoodriverheather** [comment]
@yuliujpl Great! So, if CollectionLevelFile is true, then CollectionLevelFileHierarcy needs to contain the "path" for the CollectionLevelFile.  For example, a Collection Level File in the  Genome Editing  collection could be something like Genome_Editing_Consortium/Documentation. Is that correct?

**Comment by hoodriverheather** [comment]
@yuliujpl So would something like this in the filemetadata row for a collection level file. CollectionLevelFile=True and CollectionLevelFileHierarcy=Example: Genome_Editing_Consortium/Genome_Editing_Consortium/
Genome_Editing_Consortium/Benchmark

**Comment by yuliujpl** [comment]
Yup! Just one less level of Genome_Editing_Consortium! So more like:
Genome_Editing_Consortium/
Genome_Editing_Consortium/Benchmark

**Comment by hoodriverheather** [comment]
@yuliujpl Perfect! I'll add this to the metadata dictionary.

**Comment by hoodriverheather** [comment]
Done.
